### PR TITLE
Add id to products link

### PR DESCRIPTION
**Pull request by Pacane**

### CODE REVIEW

File: client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.ui.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
                                     </a>
                                 </li>
                                 <li>
-                                    <a href="#{nametokens.PRODUCTS}">
+                                    <a ui:field="productsAnchor" href="#{nametokens.PRODUCTS}/1">
```
Comment (olafleur) [inline client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.ui.xml]: why the /1 ?

Comment (Pacane) [inline client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.ui.xml]: @olafleur that's a leftover, removed!

File: client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java [hunk 2]
```diff
@@ -95,6 +97,24 @@
...
         $("body").addClass(getStyle(brand));
+
+        setProductsHref(brand);
+    }
+
+    private void setProductsHref(Brand brand) {
+        String nameToken = NameTokens.PRODUCT;
+
+        if (localeHelper.isFrench()) {
+            nameToken = NameTokens.translate(nameToken);
```
Comment (jclariviere) [inline client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java]: We could set NameTokens fields private and add getters that do this check (in another PR)

**Comment by olafleur** [comment]
LGTM

File: client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java [hunk 3]
```diff
@@ -95,6 +94,24 @@
     public void changeBrand(Brand brand) {
```
Comment (jclariviere) [inline client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java:98]: Is this always called (even without "changing brand")? If not, the href won't get set the first time.

File: client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java (hunk 2 above)
Comment (Pacane) [inline client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java]: Hm, I could also use `NameTokensConstants`

File: client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java (hunk 3 above)
Comment (Pacane) [inline client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java:98]: Yes it is

**Comment by jclariviere** [comment]
LGTM

**Pull request merged**